index.bs

```diff
@@ -1477,7 +1486,9 @@ The [=authenticator data=] structure is a byte array of 37 bytes or more, as fol
         <td>
             Flags (bit 0 is the least significant bit):
             - Bit 0: [=Test of User Presence=] (`TUP`) result.
-            - Bits 1-5: Reserved for future use (`RFU`).
+            - Bit 1: Reserved for future use (`RFU1`).
```

Comment:
Hi there, we use the ever-so-mysterious Bit 1 to indicate that the authenticator believes that the assertion was sent over a wireless transport (rather than a wired transport).